easybuild/easyconfigs/p/PyDamage/PyDamage-0.70-foss-2021a.eb

```diff
@@ -0,0 +1,65 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'PyDamage'
+version = '0.70'
+
+homepage = 'https://github.com/maxibor/pydamage'
+description = """Pydamage, is a Python software to automate the process of
+contig damage identification and estimation. After modelling the ancient DNA
+damage using the C to T transitions, Pydamage uses a likelihood ratio test
+to discriminate between truly ancient, and modern contigs originating from
+sample contamination."""
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake',    '3.20.1'),
+    ('make',     '4.3'),
+    ('Check',    '0.15.2'),
+    ('SWIG',     '4.0.2'),
+    ('expat',    '2.2.9'),
+    ('bzip2',    '1.0.8'),
+    ('zlib',     '1.2.11'),
```

Comment:
@smoretti Please only include what's actually required (probably none of these in this case)

Reply:
Right